src/theme/DocRoot/Layout/index.js

```diff
@@ -24,7 +24,7 @@ export default function DocRootLayout({ children }) {
     if (isBrowser && !hasInitialized) {
       (async () => {
         try {
-          await initOpenReplay();
+          // await initOpenReplay();
```

Comment:
This can be reverted if you'll merge `main` into your branch.

I fixed it [here](https://github.com/hasura/ddn-docs/commit/d8e07690a83aa2af1433ff85ab9b8408364fb325) after it snuck through in [this PR](https://github.com/hasura/ddn-docs/commit/183a801f96a223e641771c62fccee7b6713442d2#diff-f671806f1a322599339118296e5202d5f3632a3a188b58fc837319d74bc176c9R43) last week; I confirmed with @Varun-Choudhary before merging.

Reply:
Damn. I always forget about this one (to not commit). Will fix